EmpleoDotNet/Controllers/JobOpportunityController.cs

```diff
@@ -27,27 +28,18 @@ public ActionResult Index(JobOpportunityPagingParameter model)
         // GET: /JobOpportunity/Detail/4-jobtitle         
         public ActionResult Detail(string id)
         {
-            if (string.IsNullOrEmpty(id))
-                return RedirectToAction("Index");
-
-            var url = id.Split('-');
-
-            if (url.Length == 0)
-                return RedirectToAction("Index");
-
-            var value = 0;
-
-            if (!int.TryParse(url[0], out value))
+            var value = GetIdFromTitle(id);
```

Comment:
would it  be better if we change value with a more self explain name, like jobId ?